tests/functional/z_cluster/test_performance_profile_validation.py

```diff
@@ -128,7 +174,11 @@ def test_change_cluster_resource_profile(self, perf_profile):
             resource_name=storage_cluster_name,
             namespace=config.ENV_DATA["cluster_namespace"],
         )
-        exist_performance_profile = storage_cluster.data["spec"]["resourceProfile"]
+        try:
+            exist_performance_profile = storage_cluster.data["spec"]["resourceProfile"]
+        except KeyError:
+            exist_performance_profile = None
+            pass
```

Comment:
Please add a log message or a comment about the reseason we passed the test here.